Makefile

```diff
@@ -1,3 +1,28 @@
+###############################################################################
+###                               Single-node                               ###
+###############################################################################
+
+SINGLE_NODE_SETUP_FILE=single-node/docker-compose.yml
+
+build: stop
+	@while [ -z "$$REPO" ]; do \
+    read -r -p "Repository [evmos/ethermint]: " REPO; \
```

Comment:
there seems to be a mix of spaces and tabs as whitespace, which messes up the formatting a little bit